java/src/main/native/CMakeLists.txt

```diff
@@ -136,17 +136,20 @@ endif (RMM_INCLUDE AND RMM_LIBRARY)
 # - NVStrings -------------------------------------------------------------------------------------
 
 find_path(NVSTRINGS_INCLUDE "nvstrings"
-          HINTS "$ENV{NVSTRINGS_ROOT}/include"
+          HINTS "${PROJECT_SOURCE_DIR}/../../../../cpp/include"
+                "$ENV{NVSTRINGS_ROOT}/include"
                 "$ENV{CONDA_PREFIX}/include/nvstrings"
```

Comment:
Should this include path remain?  Seems like we would not want to pick up anything related to NVStrings from outside the project repo now that it's part of the repo.

Reply:
I thought it best to leave them, but after looking through the cpp/CmakeLists.txt I think you are right.

Reply:
@jlowe I addressed your change by removing NVSTRINGS_INCLUDE, as it is covered by CUDF_INCLUDE.